Jellyfin.Api/Models/NotificationDtos/NotificationDto.cs

```diff
@@ -0,0 +1,51 @@
+using System;
```

Comment:
```suggestion
#nullable enable

using System;
```
Why didn't you add this to these files?

Reply:
I didn't see a need for it just because all of the files which these notifications would be built from in `MediaBrowser.Model` for notifications don't have nullable reference types. But I suppose they will eventually, so I'll update that.

Reply:
The plan is to enable nullable reference types everywhere, so all new files should already enable it

Reply:
Makes sense, updated!